### ISSUE TITLE
Use civicplatform.org as the canonical domain for the 2018 frontend container

### ISSUE THREAD

**Issue by DingoEatingFuzz** [hackoregon/civic-devops#172]
Since this year we are telling the national Civic story, the name `civicpdx` is no longer accurate. We now own the domain `civicplatform.org` that we would like to use instead.

In the short term, what this means is making what is currently at `staging-2018.civicpdx.org` available at `civicplatform.org` and ideally making `staging-2018.civicpdx.org` redirect to the new URL.

Although we don't want to go live too early, we want to be confident that as soon as we are ready to go live, it's as simple as flipping a switch.

**Comment by iant01** [comment]
The easiest and safest right now for this change may be to create a dns entry that points to the load balancer (like staging-2018.civicpdx does and create and spinup yet another service (2018.civicplatform ?) That uses the same container image as the staging-2018 service so that there is a non conflicting listener group for it and proper target group. That way both will coexist and function. The key part would be making a change to the travis build for staging-2018 to do an extra deploy for the new 2018.civicplatform service using the same container image. This way when staging is updated the both get deployed and upgraded to remain in sync. Then at some point change back to a single service deploy and get rid of the staging-2018 naming and remove the old images, service/task definitions, etc. .

**Comment by MikeTheCanuck** [comment]
@DingoEatingFuzz Who has control of the `civicplatform.org` domain?  

Who can either get the **NS** records pointed at AWS, or can create **A** (or **CNAME**?) record to direct requests for http://civicplatform.org over to our ALB?

If we're going to use this in seven days' time, we need to get one of these two changes made ASAP to be able to get the AWS side of things testing.

**Comment by DingoEatingFuzz** [comment]
The domain ceremony is now done. I have access to the domain.

The A record is an alias to the ALB, and I created a new listener rule by hand to route civicplatform.org to the Civic2018 service/container. 

I back-filled those changed I made by hand to the CF templates, but I haven't yet applied those changes to the stack. I didn't want to do that without some eyes on it: https://github.com/hackoregon/hackoregon-aws-infrastructure/pull/39/files
